<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #Background{
background-color:rgb(102, 84, 94)}
</string>
    </property>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>330</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>23</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 23pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
     </property>
     <property name="text">
      <string>FirstName LastName</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="logout_button">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>720</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(102, 84, 94);
border-radius: 20px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Resource/logout.png</normaloff>../Resource/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>20</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="avatar">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(102, 84, 94);
border-radius: 20px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>E:/Tam/Programming/software/Message_Platform/client/desktop/GUI/Resource/default_avatar.png</normaloff>E:/Tam/Programming/software/Message_Platform/client/desktop/GUI/Resource/default_avatar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>160</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="deadline_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>600</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(246, 224, 181);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Manage Deadline</string>
     </property>
    </widget>
    <widget class="QPushButton" name="member_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>510</y>
       <width>600</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(246, 224, 181);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Manage Member</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource/facebook.qrc"/>
 </resources>
 <connections/>
</ui>
